### PR TITLE
Add View Option for raw/unadjusted trace

### LINKED ISSUE #152: Add View Option for raw/unadjusted trace
Per https://github.com/jaegertracing/jaeger/issues/617, the backend will support `/api/traces/{traceID}/?raw=true` URL. We should add it to the View Options dropdown as "Trace JSON (unadjusted)".

Also, `s/View Trace JSON/Trace JSON/`

image

### PR DESCRIPTION
**Pull request by yurishkuro**
Fixes #152

### CODE REVIEW

File: src/components/TracePage/TracePageHeader.js [hunk 1]
```diff
@@ -104,7 +104,12 @@ export default function TracePageHeader(props: TracePageHeaderProps) {
               <Dropdown.Menu>
                 <Dropdown.Item>
                   <a rel="noopener noreferrer" target="_blank" href={`/api/traces/${traceID}`}>
```
Comment (yurishkuro) [inline src/components/TracePage/TracePageHeader.js]: @tiffon I don't see this file in #151, it probably needs adjustment with the prefix as well

**Comment by black-adder** [comment]
add this to the change log so it can go out with 1.1?

File: src/components/TracePage/TracePageHeader.js (hunk 1 above)
Comment (tiffon) [reply src/components/TracePage/TracePageHeader.js]: @yurishkuro Yes, this needs the `prefixUrl(...)`.

Also, is the trailing `/` in the raw version URL intentional?

```
/api/traces/${traceID}/?raw=true
```

Seems like the only difference should be the query string is appended?

```
/api/traces/${traceID}?raw=true
```

**Comment by tiffon** [comment]
@yurishkuro The `prefixUrl(...)` needs to be in master in the event of `package.json#homepage` is changed; it's not just relevant to the query-config PR (#151). 

I went ahead and added it (and fixed a failing unit test).

**Pull request merged**